test/EventListener/JsonApiProblemExceptionListenerTest.php

```diff
@@ -63,10 +63,9 @@ public function it_runs_on_json_route_formats(): void
         $listener = new JsonApiProblemExceptionListener($this->exceptionTransformer->reveal(), false);
         $this->request->getRequestFormat()->willReturn('json');
         $this->request->getContentType()->willReturn(null);
+        $listener->onKernelException($this->event);
 
-        $this->event->setResponse(Argument::type(JsonResponse::class))->shouldBeCalled();
-
-        $listener->onKernelException($this->event->reveal());
+        $this->assertNotNull($this->event->getResponse());
```

Comment:
The original was testing the instance of the returned value. Seems better to validate it similarly

Reply:
Fixed this, it now does an extra assert for to check the content of the event